runtime/codert_vm/thunkcrt.c

```diff
@@ -55,10 +55,10 @@
 /* Encoding format is:
  *
  * 1 byte			argCount
- * 128 bytes		encoded bytes (up to 255 args + 1 return type)
+ * 129 bytes		encoded bytes (up to 1 receiver + 255 args + 1 return type)
  */
 
-#define J9_THUNK_MAX_ENCODED_BYTES	129
+#define J9_THUNK_MAX_ENCODED_BYTES	130
```

Comment:
Looking at this again, I'm not sure that I need to adjust this value. The receiver should be included in the 255-parameter limit in the JVM

Reply:
Yes, the receiver contributes toward the argument count for the purpose of the limit of 255. JVMS section 4.3.3:
> A method descriptor is valid only if it represents method parameters with a total length of 255 or less, where that length includes the contribution for `this` in the case of instance or interface method invocations.

So I'll change this back